tests/pytest/core/test_views.py

```diff
@@ -9,8 +9,14 @@
 def test_homepage(client):
     path = reverse("core:index")
     response = client.get(path)
-    assert response.status_code == 200
-    assert "transit" in str(response.content)
+
+    agencies = TransitAgency.all_active()
+    if agencies.count() == 1:
+        assert response.status_code == 302
+        assert response.url == agencies[0].index_url
+    else:
+        assert response.status_code == 200
+        assert "transit" in str(response.content)
 
```

Comment:
I think we do want a new test here, since this is introducing new behavior.

`TransitAgency.all_active()` can be mocked in one case to return just a single entry, to test this change.

Reply:
I hadn't thought about mocking `TransitAgency.all_active()`. Nice! I'll work on making that change.